internal/runners/shell/shell.go

```diff
@@ -87,7 +90,9 @@ func (u *Shell) Run(params *Params) error {
 	}
 
 	if process.IsActivated(u.config) {
-		return locale.NewInputError("err_shell_already_active", "", proj.NamespaceString(), proj.Dir())
+		activatedProjectNamespace := os.Getenv(constants.ActivatedStateNamespaceEnvVarName)
+		activatedProjectDir := os.Getenv(constants.ActivatedStateEnvVarName)
+		return locale.NewInputError("err_shell_already_active", "", activatedProjectNamespace, activatedProjectDir)
```

Comment:
nit: do we want to have a default message in the case that one or both of these env vars are not set? While we do set them on activation the user still has the ability to change their environment.

Reply:
I'm not sure I understand your question correctly, but I don't believe we guard against any user-setting/unsetting of `ACTIVESTATE*` env variables that we make use of/depend on. If these are set, we assume it was because of us. If they are not, we assume it was because of us.

Reply:
In the case that these aren't set the message the user will get will be incomplete. My point was that we could guard against that by having a default message. This might be preferable to a message like:

```
You are already in a shell for ' ', located at ' '.
```

It's a niche case, which is why this is a nit, but it is also fairly easy for us to provide a default message in the case that these variables are not set.

Reply:
I see what you mean. Technically, that is out of scope for this PR since the error message already existed. Since Mark has not run into this in his test plan, I suspect there isn't currently a need for this. Feel free to open a task in JIRA to discuss on triage though.